src/index.html

```diff
@@ -31,6 +31,47 @@
             <a href="https://github.com/Fondem-Jr">Fondem Junior</a>
           </p>
     </footer>
+
+    <div class="reservepopup">
+      <div class="reservation container" id="reservation-container">
+        <div class="reservation-img-container">
+          <div id="reservation-img">
+          </div>
+          <span class="close-reservation"><button id="close-reserve-popup"><i class="fa fa-window-close" aria-hidden="true"></i></button></span>
```

Comment:
The style of the button doesn't seem to fit very well with the rest of the design.
![button](https://user-images.githubusercontent.com/71156605/142286925-2344c625-a1a4-4bcb-a1fc-316d2da11120.png)